WhaleSpotting/Repositories/WhaleSightingRepo.cs

```diff
@@ -90,5 +93,25 @@ public List<WhaleSightingResponse> ListApprovedSightings()
             throw new ArgumentOutOfRangeException($"No approved whale sightings in the database", ex);
         }
     }
+
+    public List<TripPlannerResponse> ListNearBySightings(float inputLat, float inputLon)
+    {
+        try
+        {
+            List<TripPlannerResponse> SighingsList = context.WhaleSightings
+             .Where(ws => (int)ws.ApprovalStatus == 1)
+             .Include(ws => ws.WhaleSpecies)
+             .Select(x => new TripPlannerResponse(x, inputLat, inputLon))
+             .AsEnumerable()
+             .OrderBy(x => x.Distance)
+             .Take(5)
+             .ToList();
```

Comment:
Can you change the indentation on these to 4 spaces?

Reply:
changed